arm-sql/2014-04-01/swagger/dataMasking.json

```diff
@@ -0,0 +1,460 @@
+                },
+                "tableName": {
+                    "type": "string",
+                    "description": "Gets or sets the table name on which the data masking rule is applied."
+                },
+                "columnName": {
+                    "type": "string",
+                    "description": "Gets or sets the column name on which the data masking rule is applied."
+                },
+                "maskingFunction": {
+                    "type": "string",
+                    "description": "Gets or sets the masking function that is used for the data masking rule."
+                },
+                "numberFrom": {
+                    "type": "string",
+                    "description": "Gets or sets the numberFrom property of the masking rule."
+                },
+                "numberTo": {
+                    "type": "string",
+                    "description": "Gets or sets the numberTo property of the data masking rule."
+                },
+                "prefixSize": {
+                    "type": "string",
+                    "description": "Gets or sets the prefixSize property that is used for the data masking rule."
+                },
+                "suffixSize": {
+                    "type": "string",
+                    "description": "Gets or sets the suffixSize property that is used for the data masking rule."
+                },
+                "replacementString": {
+                    "type": "string",
+                    "description": "Gets or sets the replacementString property that is used for the data masking rule."
+                }
+            },
+            "required": [
+                "id",
+                "ruleState",
+                "maskingFunction"
+            ],
+            "description": "Represents the properties of an Azure SQL Database data masking rule."
+        },
+        "DataMaskingRule": {
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/DataMaskingRuleProperties",
+                    "description": "Represents the properties of the resource."
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/definitions/SubResource"
+                }
+            ],
+            "description": "Represents an Azure SQL Database data masking rule."
+        },
+        "DataMaskingRuleListResult": {
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/DataMaskingRule"
+                    },
+                    "description": "Gets or sets the list of Azure Sql Database data masking rules"
+                }
+            },
+            "description": "Represents the response to a List data masking rules request."
+        },
+        "Resource": {
```

Comment:
As per the V2 version of Resource definition "sku" is a top level property of Resource.